### PR TITLE
2813 NTR visceral muscle and subclasses

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses 2813 to create new terms for visceral muscle and subclasses

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+def: "A smooth muscle tissue that is associated with internal organs (viscera)." [PMID:12132390, PMID:33729000]
+is_a: UBERON:0001135 ! smooth muscle tissue
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:32:12Z xsd:dateTime
+
+[Term]
+id: UBERON:8600006
+name: visceral striated muscle tissue
+def: "A striated muscle tissue that is associated with internal organs (viscera), except the heart." [https://pressbooks.pub/rbtallitsch/chapter/chapter-6-muscle-tissue/, PMID:8720463]
```
[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224773]: Re: dbxref: https://pressbooks.pub/rbtallitsch/chapter/chapter-6-muscle-tissue/#footnote-100-1, it mentions "noncardiac visceral striated muscle", but unsure what content is referenced to support this class, unless this class is meant to specifically exclude visceral striated muscle tissue of the heart.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+synonym: "striated visceral muscle tissue" EXACT []
+is_a: UBERON:0002036 ! striated muscle tissue
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224775]: The text def of 'striated muscle tissue' is:

"Muscle tissue that contains fibers that are divided by **transverse** bands into striations."

Should this be reserved for the other new term 'transversely striated visceral muscle tissue'?

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+is_a: UBERON:8600004 ! visceral muscle tissue
+relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date 2023-02-16T10:34:43Z xsd:dateTime
+
+[Term]
+id: UBERON:8600007
+name: transversely striated visceral muscle tissue
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224782]: For label consistency, consider labelling as 'visceral transversely striated muscle tissue' and making this a syn if necessary.

File: src/ontology/uberon-edit.obo [hunk 4]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+def: "A striated visceral muscle tissue that has a transverse striation." [PMID:8720463]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224783]: From the dbxref:
"The muscular cells of invertebrates can be divided into three major classes on the basis of their striation pattern: **transversely striated, obliquely striated**, or smooth muscle."

Is 'visceral striated muscle tissue' meant to be a grouping term for 'transversely striated visceral muscle tissue' and 'obliquely striated visceral muscle tissue'? If so, will you be adding 'obliquely striated visceral muscle tissue'?

Alternatively, consider modelling 'visceral striated muscle tissue' as 'obliquely striated visceral muscle tissue'.

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comments.

File: src/ontology/uberon-edit.obo (hunk 4 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224783]: Thank you @bvarner-ebi. From the dbxref:

"Although all the arthropod muscles present a common ultrastructural pattern, which may be described as `transversely striated muscle` with continuous Z lines, there are dissimilarities among zoological groups, and even within the same species, in relation to the anatomical type of muscle. Two anatomical types of muscles, specialised for different functions, may be distinguished in arthropods: skeletal (somatic) muscles, `and visceral muscles (digestive tract and hemolymphatic vessels)`. Another type of muscle should be added in insects: the flight muscles, which may either be of the direct type (odonates, lepidopteran and orthopteran) or the indirect type (dipteran, hymenopteran, coleopteran and hemipteran)."

Meaning that there are three different transversely striated muscle, one of them being visceral. However, although not mention for arthropods, in annelids there is an `obliquely striated muscle` associated with the intestinal wall:

"Recently, this [Obliquely striated] muscle type has also been described in the intestinal wall (Fig. 7a,b) and pseudohearts (Fig. 7c,d) of the oligochaete E.foetida (Royuela et al., 1995)."

Visceral obliquely striated muscle are also [found in snails](https://doi.org/10.1007/bf00227599). Interestingly, in this reference there are cross striated muscles, which might be a synonym for transversely striated muscles.

File: src/ontology/uberon-edit.obo (hunk 2 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224775]: Thank you for spotting it, I've noticed some inconsistencies  needed for striated muscles and its modelling that probably they should be addressed in a different ticket.

File: src/ontology/uberon-edit.obo [hunk 5]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
-synonym: "retro-auricular region" EXACT [PMID:33988139]
-synonym: "retroauricular area" EXACT [PMID:29470606]
-is_a: UBERON:0000061 ! anatomical structure
-relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
-relationship: part_of UBERON:0000033 ! head
-property_value: curator_notes "Exists at least in all mammals. https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Retroauricular-region" xsd:string
-property_value: dcterms-date 2023-02-09T09:39:18Z xsd:dateTime
+id: UBERON:8600004
+name: visceral muscle tissue
+def: "A muscle tissue that is associated with internal organs (viscera), except the heart." [BTO:0002780, CLAO:0001295]
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224755]: > The BTO def asserts that the heart does have visceral muscle tissue, but it is not composed of smooth muscle fibres

Thank you for spotting this, I misunderstood the definition.

> Uberon considers bones as organs, so consider revising text definition by adding qualifiers to "internal organs" or to simply use the term "viscera".

I have included "soft internal organs" to exclude bones.

> Consider adding exact syn "visceral muscle" (BTO:0002780).

Thank you

File: src/ontology/uberon-edit.obo [hunk 6]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+comment: It is quite common to refer to visceral muscles as smooth muscles, by analogy with vertebrates; however most visceral muscles in arthropods are actually striated. {xref="PMID:12132390", xref="PMID:8720463"}
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224756]: I have changed the references. The first one is from the fly ontology where this sentence is found in the comment. In turn, it has a reference, but I do not have access to it. The second one is wikipedia, that uses visceral muscle as a synonym to smooth muscle, showing how common it has been used as a synonym.

File: src/ontology/uberon-edit.obo [hunk 7]
```diff
@@ -224736,18 +224750,40 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
+is_a: UBERON:0002385 ! muscle tissue
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224757]: I have used a subclass of 'has part': 'composed primarily of' some 'visceral muscle cell'

File: src/ontology/uberon-edit.obo (hunk 1 above)
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224773]: > Re: dbxref: https://pressbooks.pub/rbtallitsch/chapter/chapter-6-muscle-tissue/#footnote-100-1, it mentions "noncardiac visceral striated muscle", but unsure what content is referenced to support this class, unless this class is meant to specifically exclude visceral striated muscle tissue of the heart.

This reference is one of the reasons I was under the impression that cardiac muscle was not considered visceral muscle. Therefore, 'cardiac muscle tissue' is a subclass of 'visceral striated muscle tissue'.

File: src/ontology/uberon-edit.obo [hunk 8]
```diff
@@ -224736,18 +224751,53 @@ relationship: dc-contributor http://orcid.org/0000-0001-6677-8489
...
-synonym: "retroauricular area" EXACT [PMID:29470606]
-is_a: UBERON:0000061 ! anatomical structure
-relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
-relationship: part_of UBERON:0000033 ! head
-property_value: curator_notes "Exists at least in all mammals. https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Retroauricular-region" xsd:string
-property_value: dcterms-date 2023-02-09T09:39:18Z xsd:dateTime
+id: UBERON:8600004
+name: visceral muscle tissue
+def: "A muscle tissue that is associated with soft internal organs (viscera)." [BTO:0002780, CLAO:0001295]
+comment: It is quite common to refer to visceral muscles as smooth muscles, by analogy with vertebrates; however, most visceral muscles in arthropods are actually striated. {xref="https://en.wikipedia.org/wiki/Smooth_muscle", xref="FBbt:00005070"}
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224757]: PMID:8720463 seems to support this comment, perhaps more than the current dbxrefs:

"Of all invertebrate muscles, the transversely striated muscle with continuous Z lines is the most similar to the vertebrate skeletal muscle and is present in arthropods, whose musculature (including the visceral muscles) only consists of this cell type. "

**Review by bvarner-ebi — APPROVED** [review]

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Pull request merged**